docs/dev/run_the_etl.rst

```diff
@@ -181,7 +182,36 @@ add them in the Launchpad tab of the job like so::
       config:
         clobber: true
 
-**Running the PUDL ETL**
+You can also adjust the years to process for each dataset using the Launchpad
+tab::
```

Comment:
Might be helpful to mention that hovering over the name of the relevant settings on the righthand side of the launchpad shows all available options, e.g. all viable years for each dataset.

Reply:
Ah good suggestion. Just added.

Reply:
I could not for the life of me figure out how to do anything with launchpad. Maybe I need some hand-holding.